src/agent/Cargo.toml

```diff
@@ -19,3 +19,7 @@ members = [
 
 [profile.release]
 lto = "thin"
+
+[patch.crates-io]
+# TODO: remove once PR is merged upstream https://github.com/dmolokanov/appinsights-rs/pull/280
+appinsights = { git = "https://github.com/Porges/appinsights-rs", branch="audit-fixes" }
```

Comment:
Oh, I missed this: here and wherever we depend on this branch, I suggest we actually pin to the appropriate `rev`.

If we don't do this, and we merged this PR as-is, then the build could break if you pushed a breaking change to the branch (e.g. making `TelemetryClient::new()` async).